docs/usage/tables/orc.md

```diff
@@ -0,0 +1,59 @@
+# ORC
+
+This page explains how you can work with ORC tables in your Unity Catalog.
+
+Let's take a look at how we can create ORC tables in Unity Catalog.
+
+## Create ORC table
+
+Use `bin/uc table create` command with the `--format ORC` option to create a new ORC table in Unity Catalog.
+
+Use the following command to create an external ORC tables (at `path/to/storage` location) with 2 colummns: `some_numbers` and `some_letters`.
```

Comment:
Can you match this PR to the AVRO's, please? 🙏  The pages should really be identical, shouldn't they?